### PR TITLE
Update draft-terbu-sd-jwt-vc.md

### PR DESCRIPTION
**Pull request by OR13**

### CODE REVIEW

File: draft-terbu-sd-jwt-vc.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 %%%
-title = "SD-JWT-based Verifiable Credentials with JSON payloads (SD-JWT VC)"
+title = "Verifiable Credentials as JSON Web Tokens"
 abbrev = "SD-JWT VC"
```
Comment (mprorock) [inline draft-terbu-sd-jwt-vc.md:3]:
```suggestion
abbrev = "JWT-VC"
```

Comment (OR13) [reply draft-terbu-sd-jwt-vc.md:3]: ^ can we reverse to "VC-JWT" ?

**Review by mprorock — APPROVED** [review]
approve of this but see suggestion on abbrev

File: draft-terbu-sd-jwt-vc.md (hunk 1 above)
Comment (mprorock) [reply draft-terbu-sd-jwt-vc.md:3]: totally good with that

**Review by OR13 — COMMENTED** [review]
Reasoning for name change.

1. Verifiable Credential is the new part not JWT or SD-JWT.
2. This item focuses on securing Verifiable Credential with 2 existing IETF work items, so you can't start the name of the work item with 2 formats, but you can start it with the name of the thing that has 2 secured representations.
3. This is truly the vc-jwt spec, the one at W3C is focused on vc+ld+json... We should make the names as simple and clear as possible.
4. Putting JWT first is not correct, if the typ values differ in the spec and don't both use +jwt suffix (this is a variation on 2).

**Review by selfissued — APPROVED** [review]
the "with JSON payloads" wording is clunky and redundant.  All JWTs have JSON payloads.

**Review by Sakurann — CHANGES_REQUESTED** [review]
we need to make a decision whether to go #120 or #131 path. and I am personally more and more favorable towards #131. and this PR is more inline with #120 than #131.

**Comment by Sakurann** [comment]
> the "with JSON payloads" wording is clunky and redundant. All JWTs have JSON payloads.

but not all JWS, which can sign anything. this is a way to draw a line with a document in W3C which uses JWS to sign over JSON-LD.

**Comment by OR13** [comment]
Closing this PR, we will try again later.... this document should be titled vc-jwt, since it does not describe securing other content types.

**Pull request closed without merging**